lib/iris/experimental/animate.py

```diff
@@ -41,7 +41,11 @@ def animate(cube_iterator, plot_func, fig=None, **kwargs):
         object. See :meth:`iris.cube.Cube.slices`.
 
     * plot_func (:mod:`~iris.plot` or :mod:`~iris.quickplot` plot):
-        Plotting function used to animate.
+        Plotting function used to animate. Must accept the signature 
+        ``plot_func(cube, vmin=vmin, vmax=vmax, coords=coords)``.
+        :func:`~iris.plot.contourf`, :func:`~iris.plot.contour`,
+        :func:`~iris.plot.pcolor` and :func:`~iris.plot.pcolormesh`
+        all accept this signature by default.
```

Comment:
Not sure about the last bit : 
I don't think "by default" means anything in this case -- surely nothing can affect the valid function syntax ?